lib/Persistence/AbstractPersistence.php

```diff
@@ -88,16 +88,18 @@ protected static function _initialize()
                 throw new Exception('unable to create directory ' . self::$_path, 10);
             }
         }
-        $file = self::$_path . DIRECTORY_SEPARATOR . '.htaccess';
-        if (!is_file($file)) {
-            $writtenBytes = @file_put_contents(
-                $file,
-                'Require all denied' . PHP_EOL,
-                LOCK_EX
-            );
-            if ($writtenBytes === false || $writtenBytes < 19) {
-                throw new Exception('unable to write to file ' . $file, 11);
-            }
+        if ((strncmp("Apache", $_SERVER['SERVER_SOFTWARE'],6) == 0) or defined('_SERVER_APACHE')) {
```

Comment:
Thanks for you're welcome :+1: 

It's not official variable, this variable may call like this `define('_SERVER_APACHE',  'false');`

Yes, it seems interesting to put it in the configuration, but I do not know how to do it the best.